lang/cs/Org.Apache.REEF.Tests/Functional/Bridge/TestCloseTask.cs

```diff
@@ -69,6 +74,40 @@ public void TestStopTaskOnLocalRuntime()
         }
 
         /// <summary>
+        /// This test is to close a running task and enforce it to break and return after the current iteration
+        /// </summary>
+        [Fact]
+        public void TestBreakTaskOnLocalRuntime()
```

Comment:
Can you use a real IMRU task? Writing an emulation Task requires you to maintain both.